networking/multiple_networks/understanding-multiple-networks.adoc

```diff
@@ -53,6 +53,8 @@ networks in your cluster:
 
  * *ipvlan*: xref:../../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-ipvlan-object_configuring-additional-network[Configure an ipvlan-based additional network] to allow pods on a host to communicate with other hosts and pods on those hosts, similar to a macvlan-based additional network. Unlike a macvlan-based additional network, each pod shares the same MAC address as the parent physical network interface.
 
+* *vlan*: xref:../../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-vlan-object_configuring-additional-network[Configure a vlan-based additional network] is a networking feature that allows the creation of virtual interfaces on a network, enabling the segmentation of network traffic. The vlan plugin creates a vlan subinterface off a `master` interface in the host network namespace and places the vlan subinterface inside the container namespace.
```

Comment:
all other descriptions start with "to allow", please use the same for consistency. The master is not always in the host network. New feature "linkInContainer".

Something like "to allow VLAN-based network isolation and connectivity for pods ....."